src/main/java/org/spongepowered/mod/mixin/core/client/server/MixinIntegratedServer.java

```diff
@@ -70,10 +70,13 @@ public void shutdown() {
         }
 
         this.mc.addScheduledTask(() -> {
-            // Need to null check in-case this is invoked very early in login
-            if (this.mc.world != null) {
+            // Vanilla calls this, but it's completely unecessary.
```

Comment:
Why would it be unnecessary? Wouldn't this cause issues in forge clients where this could be causing client threads to hang?

Reply:
The `loadWorld(null)` call already shuts down the server and kicks the client. See the rest of the comment in the file for why it can cause issues.